app/src/main/java/com/sunasterisk/rememory/data/source/local/dao/WorkDaoImpl.kt

```diff
@@ -0,0 +1,39 @@
+package com.sunasterisk.rememory.data.source.local.dao
+
+import android.content.Context
+import com.sunasterisk.rememory.data.model.Work
+import com.sunasterisk.rememory.database.SQLiteHelper
+
+class WorkDaoImpl private constructor(context: Context) : WorkDao{
```

Comment:
truyền database thay vì context 
```suggestion
class WorkDaoImpl private constructor(helper: SQLiteHelper) : WorkDao{
```